js/app.js

```diff
@@ -13,6 +13,8 @@ var Tracker = function () {
 	this.photoArray = [];
 }
 
+var tracker = new Tracker();
```

Comment:
I have been trying to convince Sublime to allow me to change these settings for weeks now. I also get an error message when I open Sublime that Sam and I had intended to work on fixing, but didn't get around to it. I guess I have another project to add to my list!